docs/proposals/multi-arch-image-mgmt.md

```diff
@@ -0,0 +1,303 @@
+
+- **docker**:
+  - [docker buildx build](https://docs.docker.com/reference/cli/docker/buildx/build/)
+  - [docker buildx imagetool](https://docs.docker.com/reference/cli/docker/buildx/imagetools/)
+  - [docker manifest create](https://docs.docker.com/reference/cli/docker/manifest/)
+- **podman (Backed by Red Hat)**: similar with `docker manifest`, it provides `podman manifest` with subcommands to create and manipulate manifest lists and image indexes.
+- **buildah**: supports building a multi-platform image to [local filesystem](https://github.com/containers/buildah/blob/main/docs/buildah-build.1.md#building-an-image-using---output-custom-build-output.).
+- **crane(Backed by Google)**: provides a single command [crane index append](https://github.com/google/go-containerregistry/blob/main/cmd/crane/recipes.md#create-a-multi-platform-image-from-scratch) to compose an image index. `crane index filter` supports filters the image index to include only platforms that are relevant to you.
+- **regctl (Individual)** provides [regctl index add/create/delete](https://github.com/regclient/regclient/blob/main/docs/regctl.md#index-commands) to creates or manages OCI image index and manifest list, `regctl artifact put --index` supports pushing the artifact and add it to an index in a single operation. It also provides native support for local environments in all commands with the OCI Layout.
+- **manifest-tool (Individual from AWS):** create docker manifest list or OCI image index in a registry by using the [manifest-tool push command with either a YAML file describing the images to assemble or by using a series of parameters](https://github.com/estesp/manifest-tool?tab=readme-ov-file#sample-usage).
+- **skopeo (Backed by Red Hat)**: Skopeo can consume and copy image index or manifest list, but it doesn't support generating or modifying them.
+
+In addition, `docker buildx` supports building a multi-arch image using the OCI image index format. Even [Docker Official Images](https://docs.docker.com/trusted-content/official-images/) are using the OCI image index format to create multi-platform images. [Homebrew](https://github.com/orgs/Homebrew/packages) publishes all images on ghcr.io using OCI image index. 
+
+## Follow-up enhancements in the future releases
+
+- Support showing platform information of tags in formatted output:  https://github.com/oras-project/oras/issues/1547
+- Support attach annotations to image index and its child image manifest in `oras attach`: https://github.com/oras-project/oras/issues/1531
+- Add status output to show the command execution result: https://github.com/oras-project/oras/issues/1575
+- Add two alias to "oras manifest fetch": https://github.com/oras-project/oras/issues/1574
```

Comment:
nit: one alias?

Reply:
Yes. I removed this issue from the future improvement since `oras manifest show` has been addressed in this PR.